netbox/dcim/choices.py

```diff
@@ -790,7 +790,9 @@ class InterfaceTypeChoices(ChoiceSet):
     TYPE_80211AC = 'ieee802.11ac'
     TYPE_80211AD = 'ieee802.11ad'
     TYPE_80211AX = 'ieee802.11ax'
+    TYPE_80211AY = 'ieee802.11ay'
     TYPE_802151 = 'ieee802.15.1'
+    TYPE_OTHER_WIRELESS = 'other-Wireless'
```

Comment:
Raw values for the types should typically be lowercase.

```suggestion
    TYPE_OTHER_WIRELESS = 'other-wireless'
```

Reply:
@jeremystretch updated and added to constants.py.  Did generate a migration, but you said it shouldn't have?

Reply:
Disregard; the migration is expected because we're referencing the `WIRELESS_IFACE_TYPES` constant from `limit_choices_to` under the WirelessLink model. I've opened #10136 to address this, but this PR is fine as-is.